src/plots/plotGroup.ts

```diff
@@ -0,0 +1,33 @@
+module Plottable.Plots {
+  export class PlotGroup extends Components.Group {
+    public entityNearest(point: Point): PlotEntity {
+      let closestPlotEntity: PlotEntity;
+      let minDistSquared = Infinity;
+      this.components().forEach((plot: Plot) => {
+        let candidatePlotEntity = plot.entityNearest(point);
+        if (candidatePlotEntity == null) {
+            return;
+        }
+
+        let position = candidatePlotEntity.position;
+        let distSquared = Math.pow(position.x - point.x, 2) + Math.pow(position.y - point.y, 2);
```

Comment:
However, this means that this code could result in counterintuitive results. Take this example:
![pointeroverlay](https://cloud.githubusercontent.com/assets/1440449/14090550/efe5c930-f4ef-11e5-8a43-658245fce563.png)

In this scenario, the algorithm will return the circled orange `Entity` rather than the green one because it's closer, even though the query point (cursor) is inside the bar.